Sources/Web3Core/Transaction/Envelope/EIP1559Envelope.swift

```diff
@@ -245,14 +245,14 @@ extension EIP1559Envelope {
 //    }
 
     public func encode(for type: EncodeType = .transaction) -> Data? {
-        let fields: [AnyObject]
-        let list = accessList.map { $0.encodeAsList() as AnyObject }
+        let fields: [Any]
+        let list = accessList.map { $0.encodeAsList() }
 
         switch type {
         case .transaction:
-            fields = [chainID, nonce, maxPriorityFeePerGas, maxFeePerGas, gasLimit, to.addressData, value, data, list, v, r, s] as [AnyObject]
+            fields = [chainID, nonce, maxPriorityFeePerGas, maxFeePerGas, gasLimit, to.addressData, value, data, list, v, r, s]
         case .signature:
-            fields = [chainID, nonce, maxPriorityFeePerGas, maxFeePerGas, gasLimit, to.addressData, value, data, list] as [AnyObject]
+            fields = [chainID, nonce, maxPriorityFeePerGas, maxFeePerGas, gasLimit, to.addressData, value, data, list]
```

Comment:
This one has been done in #683

Reply:
I'll merge this PR with the one you mentioned when it's in development branch.